Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12.13.1
+
+WORKDIR /code
+
+COPY package.json /code/package.json
+COPY yarn.lock /code/yarn.lock
+RUN yarn install
+
+COPY . /code
```

Comment:
minor comment:
can be changed with docker multistage
https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
To avoid COPY command in dev environment